docs_src/security/Ch-APIGateway.md

```diff
@@ -111,41 +111,58 @@ The TLS certificate installed in the previous step should be among the output of
 
 ### Configuration of Adding Microservices Routes for API Gateway
 
-For the current pre-existing Kong routes, they are configured and initialized statically through configuration file specified in `security-proxy-setup` application. This is not sufficient for some other new additional microservices like application services, for example.  Thus, adding new proxy Kong routes are now possibly achieved via the environment variable, `ADD_PROXY_ROUTE`, of service `edgex-proxy` in the docker-compose file.  Here is an example:
+A standard set of routes are configured statically via the `security-proxy-setup` microservice.
+Additional routes can be added via the `EDGEX_ADD_PROXY_ROUTE` environment variable.
+Here is an example:
 
 ```yaml
-edgex-proxy:
+security-proxy-setup:
       ...
     environment:
       ...
-      EDGEX_ADD_PROXY_ROUTE: "myApp.http://my-app:56789"
+      EDGEX_ADD_PROXY_ROUTE: "app-myservice.http://edgex-app-myservice:56789"
       ...
 
 ...
 
-my-app:
+app-myservice:
    ...
-   container_name: myApp
-   hostname: myApp
+   container_name: app-myservice-container
+   hostname: edgex-app-myservice
    ...
   
 ```
 
-The value of `EDGEX_ADD_PROXY_ROUTE` takes a comma-separated list of one or more (at least one) paired additional service name and URL for which to create proxy Kong routes.   The paired specification is given as the following:
+The value of `EDGEX_ADD_PROXY_ROUTE` takes a comma-separated list
+of one or more paired additional prefix and URL
+for which to create proxy routes.
+The paired specification is given as the following:
 
     <RoutePrefix>.<TargetRouteURL>
 
-where **RoutePrefix** is the name of service which requests to create proxy Kong route and it is case insensitive; it is the docker network hostname of the service that want to add the proxy Kong route in the docker-compose file if running from docker-compose, for example, `myApp` in this case.
+where **RoutePrefix** is the base path that will be created off of the root
+of the API gateway to route traffic to the target.
+This should typically be the service key that the app uses to register
+with the EdgeX secret store and configuration provider,
+as the name of the service in the docker-compose file has
+security implications when using delayed-start services.
 
-**TargetRouteURL** is the full qualified URL for the target service, like `http://myapp:56789` as it is known on on the network
-on which the API gateway is running.  For Docker,
-the hostname should match the hostname specified in the
-`docker-compose` file.
+**TargetRouteURL** is the full qualified URL for the target service,
```

Comment:
```suggestion
**TargetRouteURL** is the fully qualified URL for the target service,
```

Reply:
Fixed in https://github.com/bnevis-i/edgex-docs/commit/2fff06e6ba4522a0ffa6a91f3b1dc29a138b3c95